### PR TITLE
[Spritelab] Cap makeNumSprites at 500

### PR DESCRIPTION
**Pull request by ajpal**
Follow up to https://codedotorg.atlassian.net/browse/STAR-1999
I fixed the student's project but we should prevent this situation from happening again by just capping the number of sprites that can be created using that block.

### CODE REVIEW

File: apps/src/p5lab/spritelab/commands/spriteCommands.js [hunk 1]
```diff
@@ -2,6 +2,11 @@ import {commands as locationCommands} from './locationCommands';
 import {commands as behaviorCommands} from './behaviorCommands';
 import * as utils from '@cdo/apps/p5lab/utils';
 
+// Big numbers in some blocks can cause performance issues. Combined with live-preview,
+// this results in hanging the tab and students unable to edit their blocks. We should
+// guard against this by silently capping numbers where needed.
+const BIG_NUMBER_GUARD = 500;
```
Comment (maddiedierker) [inline apps/src/p5lab/spritelab/commands/spriteCommands.js:8]: is it possible to cap this at the block level? we likely still want a guard here, but i'm wondering if it's even possible for an individual block to have this guard as well

Comment (ajpal) [reply apps/src/p5lab/spritelab/commands/spriteCommands.js:8]: You mean, like in the blockly block itself? That's the halting problem 😜 
We can't tell what the number will be statically, the only way we can guard against this is at runtime.

Comment (ajpal) [reply apps/src/p5lab/spritelab/commands/spriteCommands.js:8]: example showing why we can't determine this statically: 
image

Comment (maddiedierker) [reply apps/src/p5lab/spritelab/commands/spriteCommands.js:8]: that was my assumption, but i was wondering if there's a point in our code where `i` has been evaluated, the block is about to be executed, and we can guard against large `i` values. it sounds like there isn't, which is why we have to guard at the command level -- that's the first place in our code where we have the actual value of `i`

**Review by maddiedierker — APPROVED** [review]

**Pull request merged**